ykrt/try_passes.py

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/env python3
+
+import argparse
```

Comment:
Conventionally it's nicer to merge imports (so long as they don't make the line too long). So `import argparse, math, os, shutil, sys, time` and then the `from` lines. Also try and sort everything alphabetically -- it helps maintenance over time.